baksnapperd.sh

```diff
@@ -131,7 +134,7 @@ case "$1" in
         if ! [ ${#snapshots[@]} -eq 0 ]; then
             ln -sfn "${snapshots[-1]}" "$1/latest-tmp"
             mv -T "$1/latest-tmp" "$1/latest"
-        elif [[ -e "$1/latest" ]]
+        elif [[ -h "$1/latest" ]]
```

Comment:
Can you break this out into a separate commit? Instead of having it hide in the `add credit` commit.
Simplest is to just remove it and squash that into `add credit` as you've done with fixing the credit.
Then add it back and commit that as a separate commit.

Thanks